source/includes/_zapier-integration.md

```diff
@@ -116,13 +116,13 @@ Familiarize yourself with the different types of users and assignables available
 * **Find Assignable by Name** Finds an existing assignable by name.
 * **Find Project by Name** Finds an existing project by name.
 
-## More details about the 10,000ft API
+## More details about the Resource Management by Smartsheet API
 
-Zapier is a convenient and powerful way to consume the 10,000ft API. The triggers, actions and searches listed above allow you to access commonly used API functionality without having to develop custom API integrations or write software code.
+Zapier is a convenient and powerful way to consume the Resource Management by Smartsheet API. The triggers, actions and searches listed above allow you to access commonly used API functionality without having to develop custom API integrations or write software code.
 
-The 10,000ft API provides access to a number of advanced features that are not currently supported with our Zapier triggers and actions. See [10,000ft API documentation](#10,000ft-api-(pro-and-enterprise-only)) for more details.
+The Resource Management by Smartsheet API provides access to a number of advanced features that are not currently supported with our Zapier triggers and actions. See [Resource Management by Smartsheet API documentation](#Resource Management by Smartsheet-api-(pro-and-enterprise-only)) for more details.
```

Comment:
We should probably check the # part. Is that supposed to link to another part of the docs?

Reply:
Good catch, missed this with the bulk renaming. This should be fixed now.